skema/program_analysis/PyAST2CAST/builtin_map.py

```diff
@@ -40,7 +40,8 @@ def build_map(filename="python_builtins.yaml"):
     skema_root = "skema/skema/program_analysis/PyAST2CAST/"
```

Comment:
I don't know if we keep track of who has what OS, but if you need testing of something particular on Windows, you can send a link or code my way.

Reply:
This particular instance is working for me on both Windows and Ubuntu, but we have run into issues with Windows pathing in the past. I think we've fixed most of those cases on the Program Analysis side of things.

 In general it would be good to use the os.path library when manipulating paths like this.

Reply:
I suggest using the Pathlib module (https://docs.python.org/3/library/pathlib.html) instead if possible - it's a bit higher-level than os.path